src/compiler/diagnosticMessages.json

```diff
@@ -1168,6 +1168,10 @@
         "category": "Error",
         "code": 2396
     },
+    "Declaration duplicates builtin global identifier '{0}'.": {

```

Comment:
I'd prefer a more helpful message. This is the better direction IMO.

Though, `:%s/duplicates/name conflicts` `:%s/builtin/built-in`. P.S. I don't know if those are valid vim replacement commands.